move2kube/application.properties

```diff
@@ -8,4 +8,5 @@ mp.messaging.incoming.kogito_incoming_stream.method=POST
 move2kube_url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
 quarkus.rest-client.move2kube_yaml.url=${MOVE2KUBE_URL:http://move2kube-svc.default.svc.cluster.local:8080}
 broker_url=${BROKER_URL:http://broker-ingress.knative-eventing.svc.cluster.local/m2k/default}
-quarkus.rest-client.notifications_yaml.url=${BACKSTAGE_NOTIFICATIONS_URL:http://host.minikube.internal:7007/api/notifications/}
\ No newline at end of file
+#quarkus.rest-client.notifications_yaml.url=${BACKSTAGE_NOTIFICATIONS_URL:http://host.minikube.internal:7007/api/notifications/}
+quarkus.rest-client.notifications_yaml.url=https://orchestrator-backstage-orchestrator.apps.cluster-xpxfb.dynamic.redhatworkshops.io/api/notifications/
```

Comment:
I guess one of them is not needed, and the latter points to a specific cluster and it's not configurable